src/dotnet-install.ps1

```diff
@@ -863,17 +882,30 @@ function DownloadFile($Source, [string]$OutPath) {
 
     $Stream = $null
 
+    $remoteFileSize = Get-Remote-File-Size -zipUri $Source
+    
     try {
         $Response = GetHTTPResponse -Uri $Source
         $Stream = $Response.Content.ReadAsStreamAsync().Result
         $File = [System.IO.File]::Create($OutPath)
         $Stream.CopyTo($File)
         $File.Close()
+
+        # Calculate the downloaded file size in bits
+        $fileSizeBytes = [long](Get-Item $OutPath).Length
+        $fileSizeBits = $fileSizeBytes * 8
+
+        # Log the downloaded file size in bits
+        Say "Downloaded file $Source size is $fileSizeBits bits."
     }
     finally {
         if ($null -ne $Stream) {
             $Stream.Dispose()
         }
+
+        if ((![string]::IsNullOrEmpty($remoteFileSize)) -and $remoteFileSize -ne $fileSizeBits) {
+            Say "The remote and local file sizes are not equal."
```

Comment:
minor: please consider outputting the sizes: source and actual.

Reply:
done